kivy/lib/sdl2.pxi

```diff
@@ -4,15 +4,15 @@
 #whose terms are available in the LICENSE file or at http://www.ignifuga.org/license
 
 
-cdef extern from "SDL_joystick.h":
+cdef extern from "SDL2/SDL_joystick.h":
```

Comment:
Without fixing `setup.py` to find it under this path everywhere by default I don't think this will be ok, nor that it's a good way as it's an API-break. Or rather a dependency-break for anyone using Kivy for longer period and compiling manually.